eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="$(MicrosoftDotNetApiCompatVersion)"/>
+  </ItemGroup>
+
+  <!-- Only enable API Compat when needed for some building assembly -->
+  <PropertyGroup Condition="'$(RunNetFrameworkApiCompat)'==''">
+    <RunNetFrameworkApiCompat>false</RunNetFrameworkApiCompat>
+    <RunNetFrameworkApiCompat Condition="'$(Net48CompatNeededProjects)'!='' 
+                                          and $(Net48CompatNeededProjects.Contains('$(MSBuildProjectName)'))">true</RunNetFrameworkApiCompat>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(RunReferenceApiCompat)'==''">
+    <RunReferenceApiCompat>false</RunReferenceApiCompat>
+
+    <RefBin>$(ArtifactsDir)bin\$(AssemblyName)\$(Configuration)\$(Platform)\$(AssemblyName).dll</RefBin>
```

Comment:
Is there no other property that can give us the value needed for `RefBin`? This seems fragile. `$(OutputAssembly)`, or `$(IntermediateAssembly)`, perhaps?

Reply:
I have to rework this whole section.  The issue is there is no good way to ensure that a hand-crafted ref assembly (or cycle-breaker) is being built after the implementation assembly.  This means the generic reverse API compat from Arcade isn't useful.

CoreFX has a separate project that does this and we may have to do that + dynamically project reference everything we need to build from that project, then call the APICompat task directly via batching.

Currently I think we should split that work off from the .NET 4.8 compat because it doesn't directly influence RC API compat and we can do it later on.  Thoughts?